.changes/next-release/feature-AWSCRTHTTPClient-5e6e380.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "feature",
+    "category": "AWS CRT HTTP Client",
+    "contributor": "",
+    "description": "Allow users to enable Post Quantum TLS via `AwsCrtAsyncHttpClient.builder().preferePostQuantumTls(true).build()`."
```

Comment:
Misspelling in `AwsCrtAsyncHttpClient.builder().preferePostQuantumTls(true).build()`. Should be `prefer`